install/kubernetes/generate-webhook-cert.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
```

Comment:
Can you make this file executable?

Reply:
Thanks, fixed. I also changed to `#!/bin/bash`.